dags/branching_local_and_remote_ampliseq.py

```diff
@@ -0,0 +1,462 @@
+import os, csv, logging, tarfile, pendulum, textwrap
```

Comment:
also wondering if we want this file? to me, these types of things, if they dont work out but you dont want to lose them, are the kinds of things you let your git commit history remember for you.